src/mlpack/bindings/R/mlpack/src/Makevars.win

```diff
@@ -1,3 +1,3 @@
-PKG_CXXFLAGS = -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=false -I. $(SHLIB_OPENMP_CXXFLAGS)
+PKG_CXXFLAGS = -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=false -I. $(SHLIB_OPENMP_CXXFLAGS) -ftrack-macro-expansion=0
```

Comment:
Wasn't the change that we only needed `-ftrack-macro-expansion=0` to enable Solaris builds, so why add it on Windows?

Reply:
It seems like it might still be necessary for Windows, but looking at the Github Actions build, it seems like the compiler is still running out of memory, so maybe `-ftrack-macro-expansion=0` is not enough?

Reply:
Ah, my bad, missed that aspect.

I have messed with -O0 and tricks like that for (large) Debian builds on small/resource-poor systems.